### PR TITLE
Fixed some errors and response implementations

### PR DESCRIPTION
**Pull request by keller00**
the fixes we  did last night with me rewriting response classes.

### CODE REVIEW

File: android/app/src/main/java/com/chorestory/app/ParentLoginActivity.java [hunk 1]
```diff
@@ -62,28 +63,45 @@ public void onClick(View v) {
...
-                            // TODO: Navigate to profile page
+                            Call<AccountResponse> accountQuery = retrofitInterface.me(tokenHandler.getToken());
+
+                            accountQuery.enqueue(new Callback<AccountResponse>() {
+                                @Override
+                                public void onResponse(Call<AccountResponse> call, Response<AccountResponse> response) {
+                                    // TODO: Navigate to profile page
+                                    navigateTo(ParentHomeActivity.class,
+                                            getResources().getString(R.string.clan_name),
+                                            response.body().getData().getClanName());
```
Comment (jimzhaozhengwang) [inline android/app/src/main/java/com/chorestory/app/ParentLoginActivity.java]: check if response is valid

Comment (keller00) [reply android/app/src/main/java/com/chorestory/app/ParentLoginActivity.java]: I just tested the scenario of the returned schema changing to simulate invalid response and the `onFailure` handler will get called in that case.

File: android/app/src/main/res/layout/fragment_parent_clan.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <android.support.v7.widget.RecyclerView
         android:id="@+id/child_recycler_view"
         android:layout_width="match_parent"
-        android:layout_height="300dp"
+        android:layout_height="450dp"
```
Comment (cr2david) [inline android/app/src/main/res/layout/fragment_parent_clan.xml]: is there a smart way to do this?

Comment (jimzhaozhengwang) [reply android/app/src/main/res/layout/fragment_parent_clan.xml]: yes, it will be changed in future PRs.

**Pull request merged**